src/url-loader/url-resolver.ts

```diff
@@ -46,8 +49,19 @@ export abstract class UrlResolver {
       url: FileRelativeUrl|PackageRelativeUrl,
       baseUrl: ResolvedUrl): ResolvedUrl {
     let resolved = urlLibResolver(baseUrl, url);
-    if (url.endsWith('/') && !resolved.endsWith('/')) {
-      resolved += '/';
+
+    // TODO(usergenic): There is no *explicit* test coverage for the missing
+    // trailing slash on urlLibResolver'd urls.  Investigate why this
+    // double-check is necessary here, since I adapted the existing logic
+    // in-place from the prior form to support urls containing search and hash.
+    // I'm guessing this is related to Windows...?
```

Comment:
Hm, maybe it's not necessary after all, it looks like urlLibResolver does preserve trailing slashes. Feel free to drop this logic then.

The tests document the expectations of this code, otherwise we'll end up scared to make any changes at all to it. No haunted graveyards! http://noidea.dog/srecon17americas2#no-haunted-graveyards-john-truscott-reese-computer-scientist-google

Reply:
+1

Reply:
Removed